src/components/Text.js

```diff
@@ -14,31 +14,21 @@ type Props = {
   markdown?: boolean,
   style?: StyleObj,
   allowFontScaling?: boolean,
-  onLayout?: Function
+  onLayout?: Function,
+  numberOfLines?: number
 };
 
-const Text = ({
-  children,
-  type,
-  markdown,
-  style,
-  allowFontScaling,
-  onLayout
-}: Props) =>
+const Text = ({ children, type, markdown, style, ...otherProps }: Props) =>
```

Comment:
you can remove children and style too, then refactor both Markdown and RnText below.